internal/gengapic/paging.go

```diff
@@ -278,6 +301,13 @@ func (g *generator) pagingCall(servName string, m *descriptor.MethodDescriptorPr
 
 func (g *generator) pagingIter(pt *iterType) {
 	p := g.printf
+	if pt.mapValueTypeName != "" {
+		p("// Holder type for string/%s map entries", pt.mapValueTypeName)
+		p("type %s struct {", pt.elemTypeName)
+		p("  Key string")
+		p("  Value %s", pt.mapValueTypeName)
```

Comment:
The value is going to be a pointer `*` too

Reply:
This encouraged me to run a test with a message value type to make sure the pointer aspect was being preserved, and it highlighted a different problem.